content/YubiHSM2/Usage_Guides/YubiHSM_2_for_Microsoft_Host_Guardian_Service:_Deployment_Guide/Prerequisites_and_Preparations.adoc

```diff
@@ -0,0 +1,27 @@
+== Prerequisites and Preparations
+
+The audience of this document is an experienced systems administrator with a good understanding of Microsoft Hyper-V virtualization management. In addition, it is helpful to be familiar with the terminology, software and tools specific to YubiHSM 2. As a primer for these, refer to link:Terminology.adoc[the Terminology chapter in this guide].
+
+In order to follow the steps provided in this guide, be sure to meet the following prerequisites:
+
+* Microsoft Windows Server 2016 or higher. The operating system should be installed in a secure computer network. The system administrator must also have elevated system privileges.
+
+* YubiHSM 2 software and tools for Windows downloaded from the link:https://developers.yubico.com/YubiHSM2/Releases/[Yubico YubiHSM 2 Release page] and available on the system to be used.
```

Comment:
mostly a nit: ideally I'd like the link to releases page to be relative rather than absolute.

Reply:
Yes indeed. Fixed.